### PR TITLE
NR-52426 Add release toolkit to pipelines and create prerelease

### PR DESCRIPTION
**Pull request by alvarocabanas**
- Adds release toolkit actions to the releaseChart action generating Changelog and Release notes.
 - Adds releaseIntegration pipeline that generates the changelog and creates the next prerelease every wednesday (tentatively).
 - Adds changelog action validation on PR workflow.
 - Release is still not implemented as we need to think how we want it to be.

### CODE REVIEW

File: .github/workflows/releaseChart.yaml [hunk 1]
```diff
@@ -28,15 +30,40 @@ jobs:
...
+        run: |
+          mv ${GITHUB_WORKSPACE}/CHANGELOG.partial.md ${GITHUB_WORKSPACE}/charts/newrelic-prometheus/RELEASE-NOTES.md
       - name: Add newrelic repository
         run: helm repo add newrelic https://helm-charts.newrelic.com
-
       - name: Release workload charts
         uses: helm/chart-releaser-action@v1.3.0
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_RELEASE_NOTES_FILE: RELEASE-NOTES.md
```
[the remark this answers is not in the document]
Comment (marcsanmi) [reply .github/workflows/releaseChart.yaml:69]: I'd say it comes from here https://github.com/newrelic/newrelic-prometheus-configurator/pull/89/files#diff-d298aa52ba694a02cd8b34918926b95a30d2ce0c599ed25547c856646507c030R62.

File: .github/workflows/releaseIntegration.yaml [hunk 2]
```diff
@@ -1,99 +1,49 @@
-name: Release prometheus-configurator image
+name: Release prometheus-configurator
 
 on:
-  release:
-    types: [prereleased, released]
+  schedule:
+    - cron: "0 4 * * 3"
```
[the remark this answers is not in the document]
Comment (paologallinaharbur) [reply .github/workflows/releaseIntegration.yaml:5]: moreover, if many repos are triggered at the same time I would be a bit scared of the result, should we randomize it somehow?

**Review by gsanchezgavier — CHANGES_REQUESTED** [review]
as we discussed , it would be good to agree on the release flows before changing the pipeline

**Comment by gsanchezgavier** [comment]
closed in favor of #126

**Pull request closed without merging**